docs/tutorial/Pre-requisites of running a Waku node.md

```diff
@@ -0,0 +1,116 @@
+---
+title: 'Pre-requisites of running a Waku node'
+---
+
```

Comment:
Thanks for following Vac rfc style, but it is not needed for nim-waku tutorials. You can remove this part. 
```suggestion

```